<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CredentialsWindow</class>
 <widget class="QDialog" name="CredentialsWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>577</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font/>
  </property>
  <property name="windowTitle">
   <string>Expense Tracker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/expense_tracker_sm.png</normaloff>:/icons/expense_tracker_sm.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255,255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="labelHeading">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(51, 51, 51);
background-color: rgb(201, 219, 220);</string>
     </property>
     <property name="text">
      <string>&lt;h2&gt;Change Password
</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelMessage">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(241, 42, 42);</string>
     </property>
     <property name="text">
      <string>text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>25</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelCurrentPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#3465a4;&quot;&gt;Password&lt;/span&gt;&lt;span style=&quot; color:#ef2929;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="buddy">
        <cstring>editCurrentPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editCurrentPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="statusTip">
        <string>username should be 4 to 12 characters long</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(254, 255, 211);
color: rgb(51, 51, 51);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Enter current pssword</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelNewPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#3465a4;&quot;&gt;New Password&lt;/span&gt;&lt;span style=&quot; color:#ef2929;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="buddy">
        <cstring>editNewPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editNewPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="statusTip">
        <string>username should be 4 to 12 characters long</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(254, 255, 211);
color: rgb(51, 51, 51);</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Create new pssword</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelConfirmPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(32, 74, 135);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#3465a4;&quot;&gt;Confirm Password&lt;/span&gt;&lt;span style=&quot; color:#ef2929;&quot;&gt;*&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="buddy">
        <cstring>editConfirmPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editConfirmPassword">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(254, 255, 211);
color: rgb(51, 51, 51);</string>
       </property>
       <property name="maxLength">
        <number>12</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm new password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>25</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(10, 145, 145);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&amp;Back</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../src/icons/back.png</normaloff>../src/icons/back.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSubmit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>40</height>
        </size>
       </property>
       <property name="font">
        <font/>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(10, 145, 145);
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>&amp;Submit</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../src/icons/ok.png</normaloff>../src/icons/ok.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>editCurrentPassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CredentialsWindow</receiver>
   <slot>onTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editNewPassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CredentialsWindow</receiver>
   <slot>onTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBack</sender>
   <signal>clicked()</signal>
   <receiver>CredentialsWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSubmit</sender>
   <signal>clicked()</signal>
   <receiver>CredentialsWindow</receiver>
   <slot>onSubmit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editConfirmPassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CredentialsWindow</receiver>
   <slot>onTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChange()</slot>
  <slot>onSubmit()</slot>
 </slots>
</ui>
